src/components/UsageDetails/index.js

```diff
@@ -23,7 +23,7 @@ const shouldShowUpgrade = (plan, appCount) => {
 	if (plan === 'free') {
 		return { show: true };
 	}
-	if (plan === 'growth') {
+	if (plan === 'growth' || plan === 'bootstrap') {
```

Comment:
@jyash97 We need to show the usage details for bootstrap users too.
Why did you set it to `false`?

Reply:
@bietkul This condition is only affecting whether to show the upgrade now button.